doc/Administration-guide/System-Configuration/General-configuration/index.md

```diff
@@ -4,4 +4,87 @@ title: General configuration
 permalink: /:path/
 sort_idx: 10
 ---
-Kaa node configuration. Inter-node communication and networking config.
+
+{% include variables.md %}
+
+* TOC:
+{:toc}
+
+# Server configuration
+
+## Public host/port configuration
+
+Kaa allows you to choose what host/port to use for internal and external communication with bootstrap and operations servers.
```

Comment:
We need to add introduction here in order to explain what is "internal" and "external" communication. Actually this is incorrect. There is bind interface and address where netty binds and serves requests and public interface which is reported to the client in form of embedded configuration in generated SDK. Need to explain why this is useful (public and private IP in cloud provider (AWS, Google Cloud, etc))